### PR TITLE
Update Intel

### LINKED ISSUE #4747: Update Intel
**Brand Name:** Intel
**Official resources for icon and color:** SVG in header of https://www.intel.ie/

Intel rebranded in September so we'll need to update our icon; some digging will be needed to see if that pesky registered trademark symbol is still needed.

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Intel

**Issue:** Closes #4747
**Alexa rank:** [806](https://www.alexa.com/siteinfo/intel.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Had to redraw the circles of the registered trademark symbol.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3548,7 +3548,7 @@
         {
             "title": "Intel",
             "hex": "0071C5",
-            "source": "https://www.intel.com"
+            "source": "https://www.intel.ie/"
```
Comment (adamrusted) [inline _data/simple-icons.json:3551]: Is it worth keeping this as `intel.com` to keep the 'global' website listed as source?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon looks perfect - just the one quick change regarding source, if you think it's appropriate?

File: _data/simple-icons.json (hunk 1 above)
Comment (PeterShaggyNoble) [reply _data/simple-icons.json:3551]: No harm in using a localised site but also no harm in leaving it as-is - in this case, I didn't want to assume the same SVG was available at intel.com. Updating now ...

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PeterShaggyNoble 💯

**Pull request merged**